### PR TITLE
Add new volunteer Jay Kumar Gupta

### PR DESCRIPTION
**Pull request by jayk-gupta**
I want to be volunteer.

### CODE REVIEW

File: data/users/jayk-gupta.json [hunk 1]
```diff
@@ -0,0 +1,14 @@
+{
+  "name": "Jay Kumar Gupta",
+  "bio": "Frontend developer ",
+  "image": "https://avatars.githubusercontent.com/u/100681165?s=400&u=3b57ae3129b51609a84806dcc1431aea2f93b074&v=4",
+  "categories": [
+    {
+      "slug": "programming |web-development",
```
Comment (codewithvoid) [inline data/users/jayk-gupta.json:7]: You need to pick 1 slug per entry at a time. Also split DSA to "Data structures" and "Algorithms"
```
  "categories": [
    {
      "slug": "programming",
      "topics": ["JavaScript", "Data structures", "Algorithms"]
    }
  ],
```

**Comment by codewithvoid** [comment]
Thanks @jayk-gupta for volunteering.

File: data/users/jayk-gupta.json [hunk 2]
```diff
@@ -0,0 +1,14 @@
...
+  "bio": "Frontend developer ",
+  "image": "https://avatars.githubusercontent.com/u/100681165?s=400&u=3b57ae3129b51609a84806dcc1431aea2f93b074&v=4",
+  "categories": [
+    {
+      "slug": "web-development",
+      "topics": ["Javascript","Data structures","Algorithms"]
+    }
+  ],
+  "contacts": {
+    "twitter": "@jayk_gupta"
```
Comment (codewithvoid) [inline data/users/jayk-gupta.json:12]: @ prefix isn't required

File: data/users/jayk-gupta.json [hunk 3]
```diff
@@ -0,0 +1,14 @@
+{
+  "name": "Jay Kumar Gupta",
+  "bio": "Frontend developer ",
+  "image": "https://avatars.githubusercontent.com/u/100681165?s=400&u=3b57ae3129b51609a84806dcc1431aea2f93b074&v=4",
+  "categories": [
+    {
+      "slug": "web-development",
+      "topics": ["Javascript","Data structures","Algorithms"]
```
Comment (codewithvoid) [inline data/users/jayk-gupta.json:8]: nit: JavaScript

File: data/users/jayk-gupta.json [hunk 4]
```diff
@@ -0,0 +1,14 @@
+{
+  "name": "Jay Kumar Gupta",
+  "bio": "Frontend developer ",
```
Comment (codewithvoid) [inline data/users/jayk-gupta.json:3]: fix spacing and casing here 
```json
  "bio": "Frontend Developer",
```

**Comment by codewithvoid** [comment]
@jayk-gupta can you address the comments please.

**Comment by jayk-gupta** [comment]
Am I a volunteer now?
I have learnt now how to do stuff and added discord link and solved the issue.
I want to be volunteer, please add me.

**Comment by jayk-gupta** [comment]
Kindly assign me the task which I need to do.

**Pull request closed without merging**